services/harmony/app/util/cmr.ts

```diff
@@ -873,6 +892,7 @@ export enum cmrQueryType {
   COLL_JSON = 'coll_json',
   COLL_UMM = 'coll_umm',
   GRID = 'grid',
+  PERMISIONS = 'permissions',
```

Comment:
Typo of `PERMISSION` and nit: For consistency, use `PERMISSION` rather than plural form.

Reply:
Fixed